src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/ContractNameServices.cs

```diff
@@ -68,7 +72,7 @@ internal static string GetTypeIdentity(Type type, bool formatGenericName)
                     typeIdentity = typeIdentityStringBuilder.ToString();
                 }
 
-                Assumes.IsTrue(!string.IsNullOrEmpty(typeIdentity));
+                //Assumes.IsTrue(!string.IsNullOrEmpty(typeIdentity));
```

Comment:
please explain this :)

Reply:
this line i commented and forgot to return to. 😆